File: 프로그래머스/0/181883. 수열과 구간 쿼리 1/수열과 구간 쿼리 1.java
class Solution {
    public int[] solution(int[] arr, int[][] queries) {
        for(int j=0; j<queries.length; j++){
            for(int i= queries[j][0] ; i<=queries[j][1]; i++){
                    arr[i]++;
            }
        }
        return arr;
    }
}

File: 프로그래머스/0/181883. 수열과 구간 쿼리 1/수열과 구간 쿼리 1.js
function solution(arr, queries) {
    for(let i=0; i<arr.length; i++){
        for(let j=0; j<queries.length; j++){
            if(queries[j][0] <= i && i <= queries[j][1]){
                arr[i]++;
            }
        }
    }
    return arr;
}